internal/commands/push.go

```diff
@@ -21,10 +21,28 @@ import (
 	"github.com/docker/docker/pkg/term"
 	"github.com/docker/docker/registry"
 	"github.com/morikuni/aec"
+	ocischemav1 "github.com/opencontainers/image-spec/specs-go/v1"
 	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 )
 
+// Type aliases to clarify to which annotation the values belong
+type dockerAppFormatValue = string
```

Comment:
Can you try this without the type aliasing? Just standard types?

Reply:
Done